cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/JwtTokenUtilService.java

```diff
@@ -8,7 +8,7 @@
 import java.util.Date;
 import java.util.function.Function;
 import javax.crypto.spec.SecretKeySpec;
-import org.apache.commons.codec.binary.Base64;
+import org.apache.tomcat.util.codec.binary.Base64;
 import org.springframework.beans.factory.InitializingBean;
```

Comment:
I was wondering why the decision to move from commons to tomcat.util?

Reply:
this is because of spring boot upgrade https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#apache-httpclient-in-resttemplate